core-web/apps/dotcms-ui/src/app/view/directives/dot-field-required/dot-field-required.directive.ts

```diff
@@ -0,0 +1,20 @@
+import { Directive, ElementRef, Input, OnInit, Renderer2 } from '@angular/core';
+import { FormGroupDirective } from '@angular/forms';
+
+@Directive({
+    selector: '[dotFieldRequired]'
+})
+export class DotFieldRequiredDirective implements OnInit {
+    @Input() for: string;
+
+    constructor(
+        private el: ElementRef,
+        private fg: FormGroupDirective,
+        private renderer: Renderer2
+    ) {}
+    ngOnInit() {
+        if (this.fg?.form.controls[this.for]?.errors?.required) {
```

Comment:
Also, I think it's a good idea not to use `for` as a `@Input()` since is a default attribute of the `HTML` label tag. We can use something more descriptive like `controlName` and add a warning in case it is not sent.

```suggestion
        if (!this.controlName) {
         console.warn("'controlName' input is needed to determine if control is required.");
         }
```

Reply:
Noted